src/DependencyInjection/CushonHealthExtension.php

```diff
@@ -50,6 +50,15 @@ public function load(array $configs, ContainerBuilder $container): void
 
         $container->registerForAutoconfiguration(DependencyCheck::class)
             ->addTag($dependencyCheckTag);
+
+        $errorStatus = 500;
```

Comment:
I can't recall if the bundle uses Symfony's Response object, but if so, could we change this to use the appropriate `HTTP_` constant? Also we may want to change the config to similarly use it?

```yaml
    cushon_health:
        error_response_code: !php/const Symfony\Component\Response::HTTP_BUSTED
```

(Or similar, I don't have the exact class to hand atm)